docs/mkdocs/docs/index.md

```diff
@@ -1,387 +1,495 @@
-![](images/FullWithBorder.png)
+ ![](images/FullWithBorder.png)
 
 ## What is ArcticDB?
 
-ArcticDB is an embedded/serverless database engine designed to integrate with Pandas and the Python Data Science ecosystem. ArcticDB enables 
-you to store, retrieve and process DataFrames at scale, backed by commodity object storage (S3-compatible storages and Azure Blob Storage).
+ArcticDB is a serverless DataFrame database engine designed for the Python Data Science ecosystem. 
+
+ArcticDB enables you to store, retrieve and process DataFrames at scale, backed by commodity object storage.
 
 ArcticDB requires *zero additional infrastructure* beyond a running Python environment and access to object storage and can be **installed in seconds.**
 
 ArcticDB is:
 
-- **Fast**: ArcticDB is incredibly fast, able to process millions of (on-disk) rows a second, and is very easy to install: `pip install arcticdb`!
-- **Flexible**: Supporting data with and without a schema, ArcticDB is also fully compatible with streaming data ingestion. The platform is bitemporal, allowing you to see all previous versions of stored data.
-- **Familiar**: ArcticDB is the world's simplest database, designed to be immediately familiar to anyone with prior Python and Pandas experience.
-
-#### What is ArcticDB _not_?
-
-ArcticDB is designed for high throughput analytical workloads. It is _not_ a transactional database and as such is not a replacement for tools such as PostgreSQL.
+- **Fast**
+    - Process millions of rows per second
+    - Quick and easy to install: `pip install arcticdb`
+- **Flexible**
+    - No data schema is needed
+    - Supports streaming data ingestion
+    - Bitemporal - stores all previous versions of stored data
+    - As easy to get started as using files
+    - Scales from dev/research to production environments
+- **Familiar**
+    - ArcticDB is the world's simplest database
+    - Easy to learn for anyone with Python and Pandas experience
+    - Just you and your data - the cognitive overhead is very low.
+
+#### Transactions: Are They Overrated?
```

Comment:
The following sound better to me:
#### Transactions: Are They Overrated?
(This can be omitted)- Transactions can be expensive and slow
- Most analytical workflows can be constructed to run without relying on transactions
- So why pay the cost of transactions when they are often not needed?
- ArcticDB doesn't support transactions because it is designed for high throughput analytical workloads

Reply:
I too have issue with this. 
* `Transactions: Are They Overrated?` the heading sounds a bit harsh. Almost like we're attacking a well established concept that's been in the software industry for years.
* We start with a question and don't give clear answer

Reply:
My take: Atomicity, Consistency and Durability are not overrated.  Isolation is.  It would be helpful to explain a little.